CMakeLists.txt

```diff
@@ -575,9 +575,11 @@ ctk_app_option(ctkDICOMRetrieve
                "Build the DICOM example application" OFF
                CTK_ENABLE_DICOM AND CTK_BUILD_EXAMPLES)
 
+if(CTK_QT_VERSION VERSION_GREATER "4")
```

Comment:
Instead of silently dropping an option if requirements are not satisfied it would be more clear to bail out with an error message if ctkDICOMQueryRetrieve is enabled and CTK_QT_VERSION<5 and let the user decide how to proceed (upgrade Qt or disable ctkDICOMQueryRetrieve).